addon/components/polaris-data-table.js

```diff
@@ -482,6 +482,12 @@ export default Component.extend(
       this.addEventHandlers();
     },
 
+    didUpdateAttrs() {
```

Comment:
I wonder if we're better off using `didRender` rather than `didUpdateAttrs` - reason being that we could potentially be passing objects to represent content (e.g. a hash of `componentName` and `props`); if the content changes but the object reference stays the same, I think `didUpdateAttrs` won't trigger, but `didRender` will.

Reply:
Ohh good call. I'll update 👍